### PR TITLE
(BIDS-2586) Use correct formatting function for total balance

### PR DESCRIPTION
**Pull request by D13ce**
This PR causes the total balance on the dashboard to use the correct formatting function.

### CODE REVIEW

File: handlers/common.go [hunk 1]
```diff
@@ -318,8 +318,7 @@ func GetValidatorEarnings(validators []uint64, currency string) (*types.Validato
 	earnings.LastWeekFormatted = utils.FormatIncomeClEl(earnings.Income7d, currency)
 	earnings.LastMonthFormatted = utils.FormatIncomeClEl(earnings.Income31d, currency)
 	earnings.TotalFormatted = utils.FormatIncomeClEl(earnings.IncomeTotal, currency)
-	// earnings.TotalChangeFormatted = utils.FormatIncome(income.ClIncomeTotal+currentDayClIncome+int64(totalDeposits), currency, true)
-	earnings.TotalBalance = utils.FormatElCurrency(totalBalance, currency, 5, true, false, false)
+	earnings.TotalBalance = "<b>" + utils.FormatClCurrency(totalBalance, currency, 5, true, false, false) + "</b>"
```
Comment (Eisei24) [inline handlers/common.go:321]: **question:** Should the text be changed to bold?
It is not bold for the balances of the individual validators.

**Review by Eisei24 — COMMENTED** [review]
It works but I have a question.

File: handlers/common.go (hunk 1 above)
Comment (D13ce) [reply handlers/common.go:321]: The dashboard on mainnet that runs on an older version has this value bold, see for example https://beaconcha.in/dashboard?validators=142321
image

I did not want to change the behavior so I added the <b>.

**Review by Eisei24 — APPROVED** [review]
lgtm

**Pull request merged**